api/src/main/java/com/github/eyefloaters/console/api/support/ErrorCategory.java

```diff
@@ -9,12 +10,15 @@
 import jakarta.ws.rs.core.Response.Status;
 import jakarta.ws.rs.core.Response.StatusType;
 
+import com.github.eyefloaters.console.api.model.Error;
 import com.github.eyefloaters.console.api.model.ErrorSource;
+import com.github.eyefloaters.console.api.model.ListFetchParams;
 
 public abstract class ErrorCategory implements Payload {
 
     private static final Map<Class<? extends ErrorCategory>, ErrorCategory> INSTANCES = Stream.of(
             new InvalidQueryParameter(),
+            new MaxPageSizeExceededError(),
             new ResourceNotFound(),
             new ServerError(),
             new BackendTimeout())
```

Comment:
IDEA gives me an interesting warning here:
`Referencing subclass BackendTimeout from superclass ErrorCategory initializer might lead to class loading deadlock`